speech/snippets/transcribe_multilanguage_gcs_beta.py

```diff
@@ -17,31 +17,30 @@
 from google.cloud import speech_v1p1beta1 as speech
 
 
-def transcribe_file_with_multilanguage_gcs(gcs_uri: str) -> str:
+def transcribe_file_with_multilanguage_gcs(audio_uri: str) -> str:
     """Transcribe a remote audio file with multi-language recognition
-
     Args:
-        gcs_uri: The Google Cloud Storage path to an audio file.
-
+        audio_uri (str): The Google Cloud Storage path to an audio file.
+            E.g., gs://[BUCKET]/[FILE]
     Returns:
-        The generated transcript from the audio file provided.
+        str: The generated transcript from the audio file provided.
     """
 
     client = speech.SpeechClient()
 
-    first_language = "ja-JP"
-    alternate_languages = ["es-ES", "en-US"]
+    first_language = "es-ES"
+    alternate_languages = ["es-ES"]
 
     # Configure request to enable multiple languages
     recognition_config = speech.RecognitionConfig(
-        encoding=speech.RecognitionConfig.AudioEncoding.LINEAR16,
-        sample_rate_hertz=16000,
+        encoding=speech.RecognitionConfig.AudioEncoding.FLAC,
```

Comment:
why change to `FLAC`? `LINEAR16` seems to give a tiny variety

Reply:
To show the function's work correctly, we need an audio file in a different language. Previous file doesn't show functionality correctly. We cannot use `.wav` file here because it requires audio in mono format. GC examples include only a stereo files in `.wav` format. 

So .`FLAC` works perfectly and demonstrates how it should work.